profile/README.md

```diff
@@ -0,0 +1,28 @@
+<img align="right" width="33%" style="margin-bottom: 2em" src="http://owlbert.io/images/owlberts-png/Popper.psd.png">
+
+# Hi, we're ReadMe! <img height=36 align=center src=https://cliply.co/wp-content/uploads/2019/06/391906110_WAVING_HAND_400px.gif>
+
+Welcome to our GitHub org! [At ReadMe][readme], we want developers to succeed. That's why building thoughtful developer experiences is at the core of everything we do.
+
+### And by “developer”, we mean a _lot_ of people…
+
+:memo:   [Anyone that builds API docs](https://readme.com/customers) on ReadMe. <br>
```

Comment:
Super nit, feel free to ignore, but this flows better if it's "Anyone who builds..." (and same for the following two lines: "people who use" and "folks who use").

Reply:
okay thank you, I was going back and forth on this lol... just updated in https://github.com/readmeio/.github/pull/10/commits/ca5be8014bbba5a85ea9e7da4a8fd6a8d016e8c2!